packer/linux/scripts/install-docker.sh

```diff
@@ -30,18 +31,22 @@ sudo curl -Lfs -o /etc/systemd/system/docker.socket https://raw.githubuserconten
 sudo systemctl daemon-reload
 sudo systemctl enable docker.service
 
-echo "Downloading docker-compose..."
-sudo curl -Lsf -o /usr/bin/docker-compose https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-Linux-x86_64
-sudo chmod +x /usr/bin/docker-compose
-docker-compose --version
+if [ "${MACHINE}" == "x86_64" ]; then
+	echo "Downloading docker-compose..."
+	sudo curl -Lsf -o /usr/bin/docker-compose https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-Linux-x86_64
+	sudo chmod +x /usr/bin/docker-compose
+	docker-compose --version
+elif [[ "${MACHINE}" == "aarch64" ]]; then
+  sudo yum install -y gcc-c++ libffi-devel openssl11 openssl11-devel python3-devel
+  sudo pip3 install docker-compose
+	docker-compose version
+fi
```

Comment:
I wonder if we want an `else` warning/error here.
Probably not important right now.

Reply:
good idea